sites/dev/src/_pages/blog/1_block-design.mdx

```diff
@@ -0,0 +1,275 @@
+
+The blocks you build will be used inside other applications; we call these "embedding application."
+
+We designed the protocol to give embedding applications control over the way blocks appear in their product. We give embedding applications the final say over how a block looks, as they have more context about the environment it lives within and how people will use it.
+
+Every Block Protocol block you build must accept a `styleVariables` object that the embedding application can pass in.
+
+This object defines a set of variables that control the basic styles of the block. We've designed these to maximise consistency and simplicity within your block. You can see the full list of these in the [styling section](#) of our documentation. Here's a rundown of the major categories:
+
+1.  Colours - a range of shades for neutrals, primary, secondary, tertiary, warning, and danger.
+2.  Typography – font family, base size, scaling ratios, and line heights
+3.  Spacing – base size and scaling ratios
+4.  Borders – weight, colours, and radius
+5.  Box shadows
+6.  Icon sizes
+
+These styling variables can do a lot of the heavy lifting to help blocks match their surrounding environment. As a block developer, you should make sure the markup in your blocks uses these styling variables to make life easier for the people using your blocks.
+
+Embedding applications can also pass a plain old CSS file into the block if they need to manually override specific styles and layouts. This should be a last resort though, as futzing with CSS selectors can get messy and may end up affecting the core functionality of the block.
+
+## Usability and UX principles
+
+No matter how beautiful a block looks, it's not worth much to anyone if it's difficult to use. Soft drop shadows and smooth gradients aren't going to delight the user if they're furiously trying to find the “save” button, and you've stashed it inside a dropdown menu that's too low contrast to see.
+
+<img
+  src="blog/1_block-design/DesignGuidelines_1.png"
+  width={900}
+  height={480}
+  alt="A text input field with a very low contrast dropdown menu below it. Inside there is a low-constrast 'save' button and a much larger and more prominent 'delte' button."
+/>
+
+Usability and user experience design (UX) is a whole discipline in its own right, and we can't cover everything in this guide. We're not expecting you to become a UX designer before you can start building blocks. However, knowing the basics of UX and some of the most important principles will go a long way in making you a better block builder.
+
+#### 1. Keep it vanilla
+
+There's a reason designers won't stop banging on about minimalism and simplicity. Adding in fancy details, decorations, or flashy animations that don't need to be there will increase the visual complexity of your block.
+It may take users longer to figure out how to use the block, or may get in the way of whatever they're trying to do with it.
```

Comment:
Do we need a new line above this one? @MaggieAppleton

Reply:
Doesn't show up in the final rendered version. The extra lines are just added by the MDX auto-formatting in VS Code.